src/kanata/linux.rs

```diff
@@ -21,6 +21,7 @@ impl Kanata {
 
         let k = kanata.lock();
         let allow_hardware_repeat = k.allow_hardware_repeat;
+        let mouse_movement_key = k.mouse_movement_key;
```

Comment:
This doesn't get updated by live reload, but this feature seems like it could reasonably be expected to be able to. To be updatable by live reload this needs to be read within the loop.

To avoid having to lock the entire Kanata struct to read this within the loop, can use a finer grained lock and have mouse_movement_key be an `Arc<Mutex<_>>`.